test/e2e-go/upgrades/application_support_test.go

```diff
@@ -286,7 +286,7 @@ func TestApplicationsUpgradeOverGossip(t *testing.T) {
 	fixture.SetupNoStart(t, filepath.Join("nettemplates", "TwoNodes100SecondTestUnupgradedProtocol.json"))
 
 	// for the primary node, we want to have a different consensus which always enables applications.
-	primaryNodeUnupgradedProtocol := consensus[consensusTestFastUpgrade(protocol.ConsensusFuture)]
+	primaryNodeUnupgradedProtocol := consensus[consensusTestFastUpgrade(protocol.ConsensusCurrentVersion)]
```

Comment:
unrelated to your change, but I don't think that this is correct anymore.
We also seen to remove application support from `primaryNodeUnupgradedProtocol` for the test to be correct.

Reply:
Yeah, that whole test isn't terribly meaningful anymore, as far as I can tell.

I just did enough to make it not fail erroneously, but it may well be the case that it's worth removing altogether.

Reply:
let me fix that in a separate ticket ? ( I'll add you to review that ).